viewer/packages/pointclouds/src/PointCloudPickingHandler.ts

```diff
@@ -36,6 +41,10 @@ export interface IntersectPointCloudNodeResult {
    * annotationId of the clicked object within a pointcloud.
    */
   annotationId: number;
+  /**
+   * externalId of clicked object within a pointcloud representing annotation.
+   */
+  instanceRef?: DMInstanceRef;
```

Comment:
Perhaps we should make it clear that this is the reference to the point cloud volume instance, and not the asset instance? And Is it just the `externalId`, or is it the `space` too?

Reply:
I think I want to add a new field to this type - something like `data`, which is a discriminated union, e.g. 
``` 
{
  type: 'cdfAnnotation';
  annotationId: number;
  assetReference?: string;
} | { 
  type: 'dmPointColudVolume';
  instanceRef: DMInstanceRef; 
};
```
Then we could deprecate the old fields. What do you think?